metacat-main/src/main/java/com/netflix/metacat/main/services/impl/ConnectorTableServiceProxy.java

```diff
@@ -0,0 +1,229 @@
+        final ConnectorRequestContext connectorRequestContext = converterUtil.toConnectorContext(metacatRequestContext);
+        service.create(connectorRequestContext, converterUtil.fromTableDto(tableDto));
+    }
+
+    /**
+     * Calls the connector table service delete method.
+     * @param name table name
+     */
+    @CacheEvict(key = "'table.' + #name")
+    public void delete(final QualifiedName name) {
+        final MetacatRequestContext metacatRequestContext = MetacatContextManager.getContext();
+        final ConnectorTableService service = connectorManager.getTableService(name);
+
+        log.info("Drop table {}", name);
+        final ConnectorRequestContext connectorRequestContext = converterUtil.toConnectorContext(metacatRequestContext);
+        service.delete(connectorRequestContext, name);
+    }
+
+
+    /**
+     *
+     * Returns table if <code>useCache</code> is true and object exists in the cache. If <code>useCache</code> is false
+     * or object does not exists in the cache, it is retrieved from the store.
+     * @param name table name
+     * @param useCache true, if table can be retrieved from cache
+     * @return table dto
+     */
+    @Cacheable(key = "'table.' + #name", condition = "#useCache")
+    public TableDto get(final QualifiedName name, final boolean useCache) {
+        final MetacatRequestContext metacatRequestContext = MetacatContextManager.getContext();
+        final ConnectorRequestContext connectorRequestContext = converterUtil.toConnectorContext(metacatRequestContext);
+        final ConnectorTableService service = connectorManager.getTableService(name);
+        return converterUtil.toTableDto(service.get(connectorRequestContext, name));
+    }
+
+    /**
+     * Calls the connector table service rename method.
+     * @param oldName old table name
+     * @param newName new table name
+     * @param isMView true, if the object is a view
+     */
+    @CacheEvict(key = "'table.' + #oldName")
+    public void rename(
+        final QualifiedName oldName,
+        final QualifiedName newName,
+        final boolean isMView
+    ) {
+        final MetacatRequestContext metacatRequestContext = MetacatContextManager.getContext();
+        final ConnectorTableService service = connectorManager.getTableService(oldName);
+
+        try {
+            log.info("Renaming {} {} to {}", isMView ? "view" : "table", oldName, newName);
+            final ConnectorRequestContext connectorRequestContext
+                = converterUtil.toConnectorContext(metacatRequestContext);
+            service.rename(connectorRequestContext, oldName, newName);
+        } catch (UnsupportedOperationException ignored) {
+        }
+    }
+
+    /**
+     * Calls the connector table service update method.
+     * @param name table name
+     * @param tableDto table object
+     */
+    @CacheEvict(key = "'table.' + #name")
+    public void update(final QualifiedName name, final TableDto tableDto) {
```

Comment:
Update table always evict cache, I thought that we don't want to evict the cache for the metadata only update.

Reply:
OK, I will move the conditional check to TableServiceImpl.